examples/why3.lp

```diff
@@ -0,0 +1,12 @@
+require open proofs.logic
```

Comment:
This file should be moved to tests/OK now.

Reply:
Done in f09160e.